net-analyzer/iplog/files/iplog-2.2.3-C23.patch

```diff
@@ -0,0 +1,36 @@
+https://bugs.gentoo.org/945194 - guard bool for modern compilers
+https://bugs.gentoo.org/712644 - type aliaces for musl
+--- a/src/iplog.h
++++ b/src/iplog.h
+@@ -21,6 +21,9 @@
+ #ifndef __IPLOG_H
+ #define __IPLOG_H
+ 
++#define _GNU_SOURCE 1
++#include <sys/types.h> /* for u_* types */
```

Comment:
This is nasty but I'll allow it in this instance.. normally we should fix to use proper C99 types instead from stdint.